### DISCUSSION TITLE
How do you use ky as a CDN without the UMD build? Is it possible?

### DISCUSSION THREAD

**Discussion by jc-camosun** [sindresorhus/ky#318] [Q&A] [answered]
I've been using ky as a CDN (there's a good reason) link and now with the UMD build dropped, I can't use it anymore in the browser. Or is there a way?

**Comment by sholladay** [comment, marked as answer]
You can still load Ky from a CDN in the browser. You just need to use ES modules to do so.

There are two ways of doing this:

1. Put your app in a `<script type="module">` tag and then `import` Ky from the CDN and use it directly. The great thing about this is that it avoids using any global variables.
    ```js
    // my-app.js
    import ky from 'https://unpkg.com/ky';
    ```

2. Alternatively, in the HTML before your app, add a separate `<script type="module">` tag that `import`s Ky and adds it to the global scope so that you don't have to modify your existing app.
    ```js
    // ky-global.js
    import ky from 'https://unpkg.com/ky';
    globalThis.ky = ky;
    ```
